mlc_config.json

```diff
@@ -99,6 +99,11 @@
       "headers": {
         "Accept-Encoding": "zstd, br, gzip, deflate"
       }
+    },
+    {
+      "headers": {
+        "User-Agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/117.0.0.0 Safari/537.36"
```

Comment:
Is this required for a specific URL?

Reply:
I've set it up to just put them for all urls. I thought it would be a good preventative measure. EOD all these URLs just have to work when any end user opens a url right?